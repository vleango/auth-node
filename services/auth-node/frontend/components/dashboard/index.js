import React from 'react';

export const DashboardPage = () => (
  <div>
    <h1>Dashboard</h1>
    <p>Welcome</p>
  </div>
);

export default DashboardPage;
