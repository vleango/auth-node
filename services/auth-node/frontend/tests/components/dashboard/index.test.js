import React from 'react';
import { shallow } from 'enzyme';
import { DashboardPage } from '../../../components/dashboard/index';

let wrapper = shallow(<DashboardPage />);

describe('Components', () => {
  describe('Dashboard', () => {
    describe('Index', () => {

      it('should correctly render DashboardPage', () => {
      	expect(wrapper).toMatchSnapshot();
      });

    });
  });
});
