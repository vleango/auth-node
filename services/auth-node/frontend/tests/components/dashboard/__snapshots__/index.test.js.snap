// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Dashboard Index should correctly render DashboardPage 1`] = `
<div>
  <h1>
    Dashboard
  </h1>
  <p>
    Welcome
  </p>
</div>
`;
